src/lando/main/migrations/0027_alter_landingjob_status_worker_type.py

```diff
@@ -2,6 +2,17 @@
 
 from django.db import migrations, models
 
+from lando.main.models.worker import WorkerType
+
+
+def set_worker_type_based_on_name(apps, schema_editor):  # noqa: ANN001
```

Comment:
This is a not completely related to this PR, but relevant to this bug. This part of the migration was erroneously dropped from the #364 during CR.

It's a bit of a moot point now, as I have fixed Dev and Prod manually, but I thought it best to have it for completeness.